tensorrt_llm/llmapi/deepseek_v32/encoding.py

```
@@ -0,0 +1,425 @@
+# copy from https://huggingface.co/deepseek-ai/DeepSeek-V3.2/blob/main/encoding/encoding_dsv32.py
+# ruff: noqa: E501
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)